src/components/profile-view/profile-view.jsx

```diff
@@ -0,0 +1,134 @@
+import { useState } from "react";
+import { Button, Card, Col, Form, Row } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import { MovieCard } from "../movie-card/movie-card";
+
+export const ProfileView = ({ movies, user }) => {
+  const storedToken = localStorage.getItem("token");
+
+  if (!user) {
+    console.log("User not found in localStorage");
+    return;
+  }
+
+  if (!storedToken) {
+    console.log("Token not found in localStorage");
+    return;
+  }
+
+  const [name, setName] = useState(user?.Name || "");
+  const [email, setEmail] = useState(user?.Email || "");
+  const [birthday, setBirthday] = useState(user?.Birthday || "");
+  const [password, setPassword] = useState(user?.Password || "");
```

Comment:
- Initially, those field should have empty values. Otherwise, the inputs will be filled with the current user data.
- Also submitting the form is failing due to an invalid date format. Check the borwser's console(network tab)

Reply:
Initially those fields are showing the existing details so that the user can see it and edit whichever fields they need. That is normally how all websites work and it seems to be a good user experience. I hope it justifies the reason of pre-filling with the data from the local storage

Also, I updated the birthday and there seems to be no error in console
![Screenshot 2024-07-03 at 8 35 06 PM](https://github.com/PoojaChinu/myFlix-client/assets/166599982/7cf4f51c-982c-469b-a8c4-9dc885d11d88)

Reply:
Failing scenario: If I update all the field except the birthdate, the submission will fail.

It not a good UX, why? because the password field will be set with the current "**encrypted password**"(totally unrecognizable in comparison to the original password) which is impossible to remember if the user submits the form without changing the password value.

Reply:
I missed that scenario, fixed now